packages/scheduler/src/forks/Scheduler.js

```diff
@@ -630,10 +616,7 @@ if (typeof localSetImmediate === 'function') {
   };
 }
 
-function requestHostCallback(
-  callback: (hasTimeRemaining: boolean, initialTime: number) => boolean,
-) {
-  scheduledHostCallback = callback;
+function requestHostCallback(callback: (initialTime: number) => boolean) {
```

Comment:
```diff
- function requestHostCallback(callback: (initialTime: number) => boolean) {
+ function requestHostCallback() {
```

Now `flushWork` is called directly but its still passed as the argument to `requestHostCallback` everywhere but its unused.